package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@metamask/safe-event-emitter": "^2.0.0",
     "async-mutex": "^0.2.6",
-    "eth-json-rpc-middleware": "^6.0.0",
+    "eth-json-rpc-middleware": "^9.0.0",
```

Comment:
This is now only directly imported in `test/util.js` - does that mean this was not actually necessary in `dependencies` and can now be moved to `devDependencies`?

Reply:
Good point.
createScaffoldMiddleware was RE-exported from [eth-json-rpc-middleware@^6](https://github.com/MetaMask/eth-json-rpc-middleware/blob/v6.0.0/scaffold.js#L2).
I think we can just use `json-rpc-engine` directly in this package (as we already import it), moving `eth-json-rpc-middleware` to `devDependencies`

Reply:
Maybe (most likely) it's totally cool and seems more sane than mixing versions like previously, just noting that this also implies switching `json-rpc-engine` from `5.4.0` to `6.1.0`.